PxWeb/Code/Api2/ModelBinder/CommaSeparatedStringToListOfStrings.cs

```diff
@@ -51,5 +41,16 @@ public Task BindModelAsync(ModelBindingContext bindingContext)
 
             return Task.CompletedTask;
         }
+
+        private static string CleanValue(string value)
+        {
+            var item2 = value.Trim();
+            if (item2.StartsWith('[') && item2.EndsWith(']'))
+            {
+                return value.Substring(1, item2.Length - 2);
+            }
+            return item2;
```

Comment:
Should "return value.Substring(1, item2.Length - 2);"  also have a Trim() ?

Reply:
good point I will add that